docs/components/card/elements.md

```diff
@@ -0,0 +1,154 @@
+## Import
+
+The toggle component allows you to render checkboxes. All you have to do is specify `type="checkbox"` for the Toggle and let the component handle the rest. Toggle is built to handle both single and multiple options.
+
+You must keep track of state yourself. The state has to be handled differently depending on whether you want to use single or multiple options. See examples below.
+
+
+```js
+import { wCard } from '@warp-ds/elements';
+```
```

Comment:
We don't need to import Warp elements this way 😊  You can check e.g. pages for Breadcrumbs and Box for how elements docs were set up. I see Buttons also have a redundant import bit in the elements docs.

Reply:
Fixed https://github.com/warp-ds/tech-docs/pull/39/commits/01519cb79fc1a1d0a8fb8272376cf7743637daf0